src/main/kotlin/no/nav/klage/service/JoarkService.kt

```diff
@@ -133,11 +128,7 @@ class JoarkService(
     }
 
     private fun getSak(klageAnkeInput: KlageAnkeInput): Sak? {
-        val isInternalFORSak = if (klageAnkeInput.innsendingsYtelseId.isNullOrBlank()) {
-            if (klageAnkeInput.tema == "FOR") {
-                klageAnkeInput.internalSaksnummer?.toIntOrNull() != null
-            } else false
-        } else if (innsendingsytelseToTema[Innsendingsytelse.of(klageAnkeInput.innsendingsYtelseId)] == Tema.FOR) {
+        val isInternalFORSak = if (innsendingsytelseToTema[Innsendingsytelse.of(klageAnkeInput.innsendingsYtelseId)] == Tema.FOR) {
             klageAnkeInput.internalSaksnummer?.toIntOrNull() != null
```

Comment:
Litt rart å forutsette at dette er `int`? Men FOR har vel tydeligvis alltid et tall som saksnummer.

Reply:
Enig i at den er litt merkelig i seg selv, men det er, som du sier, for å unngå å ha ugyldig input til FOR. Feltet er basert på input i url. Vi har jo antatt at denne stort sett kommer fra validerte lenker, men den her blir en sikring i fall noen har tukla.